modules/configuring-ide.adoc

```diff
@@ -0,0 +1,199 @@
+:_module-type: PROCEDURE
+
+[id="configuring-ide_{context}"]
```

Comment:
When you change the filename, you'll need to change its corresponding ID too.

[id="configuring-your-ide_{context}"]

Reply:
Updated.